xbmc/windows/GUIMediaWindow.cpp

```diff
@@ -727,30 +713,36 @@ bool CGUIMediaWindow::GetDirectory(const std::string &strDirectory, CFileItemLis
 // \brief Set window to a specific directory
 // \param strDirectory The directory to be displayed in list/thumb control
 // This function calls OnPrepareFileItems()
-bool CGUIMediaWindow::Update(const std::string &strDirectory, bool updateFilterPath /* = true */)
+bool CGUIMediaWindow::Update(const std::string &_arg, bool updateFilterPath /* = true */)
```

Comment:
Should we not? I tend to use it for throwaway args to make it clear it should not be used directly, rather the 'fixed' `path` below should. I guess not clear..